From dc4389ce62642209ab0e8ef6f4cae2ee43f8ce48 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Mon, 11 Dec 2023 21:12:46 +0200
Subject: [PATCH] [SW-0] gaudi3: increase bar4 size to 128gb

recovery code was ported from bootrom, on top of that the pci
bar 4 size is icreased to 128GB (nominal size).

Change-Id: I6a977fe8c9be3a4d7fd70df88d85ffbb63e7e6e9
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index bc58a25..63d254e 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -1322,6 +1322,8 @@
 
 		pcie_set_pcie_device_vendor_id();
 
+		preboot_update_bar_sizes();
+
 		if (true) { //(arcboot_rom_global.config_fuse & EFUSE_SYSCFG_PCIE_GEN_LIMIT) == 0) {
 	//		arcboot_rom_status_update(PROGRESS_ARCBOOT_ROM_PCIE_LINK_LIMIT_GEN1);
 			/* limit PCIe link gen to 1 */
